lib/xiaomi.js

```diff
@@ -518,6 +518,15 @@ const numericAttributes2Payload = (msg, meta, model, options, dataObject) => {
                 payload.battery = value;
             }
             break;
+        case '1057':
+            if (['ZNCLBL01LM'].includes(model.model)) {
+                payload = {...payload,
+                    motor_state: (options.invert_cover ? {0: 'closing', 1: 'opening', 2: 'stopped'} :
+                        {0: 'opening', 1: 'closing', 2: 'stopped'})[value],
```

Comment:
I think it should be the other way around:

```javascript
                    motor_state: (options.invert_cover ? {0: 'opening', 1: 'closing', 2: 'stopped'} :
                        {0: 'closing', 1: 'opening', 2: 'stopped'})[value],
```

because `invert_cover` **is false** by default:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/94e94bd5ff7979368722f723b05392f32bf1e07a/lib/exposes.js#L496

Reply:
Fixed.